SPECS/lua/lua.spec

```diff
@@ -152,6 +172,14 @@ install -Dpm 0644 %{SOURCE1000} $RPM_BUILD_ROOT/%{macrosdir}/macros.lua
 
 
 %changelog
+* Thu Oct 01 2020 Daniel McIlvaney <damcilva@microsoft.com> 5.3.5-11
+- Nopatch CVE-2020-24342
+- Apply patch for CVE-2019-6706 from Lua mailing list
```

Comment:
CVE-2019-6706 was already include in the new upstream Fedora spec, we can remove this entry.

Reply:
Thanks